### PR TITLE
Add a prop for controlling the items which are composed in to hidden …

### PR DESCRIPTION
**Pull request by heikkimu**
…input, when using multibrowser

### CODE REVIEW

File: src/Select.js [hunk 1]
```diff
@@ -113,7 +113,8 @@ const Select = React.createClass({
 		valueComponent: React.PropTypes.func,       // value component to render
 		valueKey: React.PropTypes.string,           // path of the label value in option objects
 		valueRenderer: React.PropTypes.func,        // valueRenderer: function (option) {}
-		wrapperStyle: React.PropTypes.object,       // optional style to apply to the component wrapper
+		wrapperStyle: React.PropTypes.object,       // optional style to apply to the component wrapper,
+		hiddenInputvalueKey: React.PropTypes.string // optional key, if you want a specific value rendered in to hidden input
```
Comment (maximzavadskiy) [inline src/Select.js:117]: capitalize Value in `hiddenInputvalueKey` ?

File: src/Select.js [hunk 2]
```diff
@@ -995,10 +996,18 @@ const Select = React.createClass({
...
+	/**
+	 * Returns the hidden input for the multivalue array
+	 * @param	{String}	name		name for inte input, compulsory
+	 * @param 	{String}	hiddenInputvalueKey key used for setting the values in to input
+	 * @param 	{String}	valueKey key used for setting the values in to input, if hiddenInputValueKey is not present
+	 */
 	renderHiddenField (valueArray) {
 		if (!this.props.name) return;
+
+		const key = (this.props.hiddenInputvalueKey ? this.props.hiddenInputvalueKey: this.props.valueKey);
```
Comment (maximzavadskiy) [inline src/Select.js:1008]: `= this.props.hiddenInputvalueKey || this.props.valueKey` ?

File: src/Select.js [hunk 3]
```diff
@@ -995,10 +996,18 @@ const Select = React.createClass({
 		}
 	},
 
+	/**
+	 * Returns the hidden input for the multivalue array
+	 * @param	{String}	name		name for inte input, compulsory
+	 * @param 	{String}	hiddenInputvalueKey key used for setting the values in to input
```
Comment (maximzavadskiy) [inline src/Select.js:1002]: Should it be added to `Select.propTypes` ?

**Review by maximzavadskiy — APPROVED** [review]
Some minor remarks, otherwise GTG!
Overall we should avoid diverging from libraries main repo, its seems like too hard job to maintain our own repos of libraries. Would be good to address that after the demo

File: src/Select.js (hunk 2 above)
Comment (heikkimu) [reply src/Select.js:1008]: 👍

File: src/Select.js (hunk 1 above)
Comment (heikkimu) [reply src/Select.js:117]: 👍

File: src/Select.js (hunk 3 above)
Comment (heikkimu) [reply src/Select.js:1002]: It is:P

**Pull request closed without merging**